spec/classes/lightblue/application/datamgmt_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+
+describe 'lightblue::application::datamgmt' do
+  let(:hiera_config){ 'spec/fixtures/hiera/hiera.yaml' }
+
+  let :facts do
+    {
+      :architecture => 'x86_64',
+      :osfamily => 'RedHat',
+      :operatingsystemrelease => '7.0'
+    }
+  end
+
+  context 'with default params' do
+    it do
+      should contain_package('lightblue-data-mgmt') \
+        .with_ensure('latest') \
+        .with_require(['Class[Lightblue::Yumrepo::Lightblue]', 'Class[Lightblue::Eap]'])
+    end
+
+    it do
```

Comment:
So should I condense these for sake of consistency? Honestly spreading it out helped me during development because I could see all of the changes that needed to be made simultaneously. I can change it if you'd like, but I don't especially feel the need to otherwise.